copying.txt

```diff
@@ -1722,7 +1722,7 @@ This copy of Python includes a copy of openssl, which is licensed under the foll
  * The implementation was written so as to conform with Netscapes SSL.
  * 
  * This library is free for commercial and non-commercial use as long as
- * the following conditions are aheared to.  The following conditions
+ * the following conditions are adhered to.  The following conditions

```

Comment:
This license is copied from the [Original SSLeay license](https://docs.oracle.com/cd/E19957-01/819-7168/gddvn/index.html). I can't find a place where the license is being maintained but it appears this typo exists across usages of the licence. It might be worth reporting upstream if you can find a maintainer of the licence.